bedita-app/views/helpers/be_html.php

```diff
@@ -132,14 +144,67 @@ public function hyphen($text, $lang = null, $excludeSelectors = array('.formula'
             }
             $text = $body->innertext;
         }
-    
+
         $text = $hyphenator->hyphenate($text, $lang);
-    
+
         foreach ($restore as $key => $value) {
             $text = str_replace('<!-- be-not-hyphen-' . $key . ' -->', $value, $text);
         }
-    
+
         return $text;
     }
 
+    /**
+     * Append timestamp to an asset to help invalidate browsers' cache.
+     *
+     * @param string $path File path.
+     * @return string
+     */
+    public function assetTimestamp($path) {
```

Comment:
@stefanorosanelli Sorry, I read your comment just now. Don't know why GitHub notifications aren't working for me. 😕 

Anyway: we don't need `BeHtmlHelper::css()` nor `BeHtmlHelper::script()`: `BeHtmlHelper` extends CakePHP's `HtmlHelper` that implements those methods. Such methods internally invoke `HtmlHelper::assetTimestamp()`, that is overridden by this PR.

Long story short: `$beHtml->script()` and `$beHtml->css()` do the work just fine. If we rename `BeHtmlHelper::assetTimestamp()` to something that fits more the scope of this PR we need to make the magic happen again by overriding `HtmlHelper::css()` and `HtmlHelper::script()` as well.